storybook/storybook-overrides.css

```diff
@@ -1,53 +1,87 @@
 /* stylelint-disable */
 
-.sbdocs-content > :is(h1, h2, h3, h4, h5, h6, p, li) {
-  color: #000000;
-  font-family: "Amsterdam Sans", Arial, sans-serif;
+.sbdocs-content > :is(h1, h2, h3, h4, h5, h6, p),
+.sbdocs-content > div > :is(h1, h2, h3, h4, h5, h6, p),
+.sbdocs-content :where(li:not(.sb-anchor, .sb-unstyled, .sb-unstyled li)) {
+  color: #000000 !important;
+  font-family: "Amsterdam Sans", Arial, sans-serif !important;
 }
 
-.sbdocs-content > :is(p, li, a),
-.sbdocs-content > :is(ol, ul) li {
-  font-size: 1rem;
-  line-height: 1.75;
+.sbdocs-content > :is(p, a),
+.sbdocs-content > div > :is(p, a),
+.sbdocs-content :where(li:not(.sb-anchor, .sb-unstyled, .sb-unstyled li)) {
+  font-size: 1.25rem !important;
```

Comment:
Yea it’s a brittle approach. Storybook theming is strangely below par and undocumented.

I still think we should go ahead with this, even if some of the texts stay small (or we could address those as well). We’re preparing to go public early 2024 and the default theme is barely legible. We should apply our own style ourselves, even as loosely as this.

Reply:
It's a tool for developers, I do not think they'll be happy with the jumbo font, you know what they think of the new font sizing. The inline code en cut off overflow icon make it look broken.

Reply:
Our design system portal is for everyone – developers, designers, product owners, business managers, suppliers, the NLDS community, other people interested in design systems. Even developers profit from reasonable line lengths. And they know how to zoom out text if they want to. I increased the vertical padding on the wrapper to prevent the link icon from being cut off on narrow screens.